app/index.html

```diff
@@ -8,9 +8,11 @@
 
     <link rel="stylesheet" href="css/windowing.css">
   </head>
-<iframe src="home.html"></iframe>
+<body>
+
```

Comment:
was there a specific reason for the new lines? if not it would cleaner to just have:
```html
<body></body>
```